### PR TITLE
Parsing CEA-608 from fmp4s post

### PR DESCRIPTION
**Pull request by ldayananda**

### CODE REVIEW

**Comment by gkatsev** [comment]
Deploy preview for *videojs-blog* ready!

Built with commit e13d78bb791fa2a99890c1c0dec6edd096405f7e

https://deploy-preview-36--videojs-blog.netlify.com

File: source/_drafts/Inband-Captions-Support-with-VHS.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  - videojs-http-streaming
+  - playback
+  - captions
+  - cea-608
+  - 608
+---
+
+With the release of [videojs-http-streaming](https://blog.videojs.com/introducing-video-js-http-streaming-vhs/) v1.2.0 on July 16th 2018, Video.js has built-in support for CEA/CTA-608 captions carried in FMP4 segments. This means that closed captions are automatically parsed out and made available to Video.js players for MPEG-DASH content and HLS streams using FMP4 segments.
+
+Caption Parsing is handled by the [mux.js]() library and interacts with videojs-http-streaming to feed parsed captions back to Video.js.
```
Comment (gkatsev) [inline source/_drafts/Inband-Captions-Support-with-VHS.md]: missing mux.js link?

File: source/_drafts/Inband-Captions-Support-with-VHS.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+tags:
+  - vhs
+  - videojs-http-streaming
+  - playback
+  - captions
+  - cea-608
+  - 608
+---
+
+With the release of [videojs-http-streaming](https://blog.videojs.com/introducing-video-js-http-streaming-vhs/) v1.2.0 on July 16th 2018, Video.js has built-in support for CEA/CTA-608 captions carried in FMP4 segments. This means that closed captions are automatically parsed out and made available to Video.js players for MPEG-DASH content and HLS streams using FMP4 segments.
```
Comment (gkatsev) [inline source/_drafts/Inband-Captions-Support-with-VHS.md]: you can do `videojs-http-streaming (VHS)` and then use VHS everywhere.

File: source/_drafts/Inband-Captions-Support-with-VHS.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  const captionParser = new CaptionParser();
+
+  // initalize the CaptionParser to ensure that it is ready for data
+  if (!captionParser.isInitialized()) {
+    captionParser.init();
+  }
+```
+
+When working with HLS and MPEG-DASH with fmp4 segments, it's likely that not all the information needed to parse out captions are included in the media segments themselves, and metadata from the `init` segment needs to be passed to the CaptionParser. For this reason, the ids of video `trak`s and timescales defined in the `init` segment should be passed into the CaptionParser.
```
Comment (gkatsev) [inline source/_drafts/Inband-Captions-Support-with-VHS.md]: `trak` -> `track`?

File: source/_drafts/Inband-Captions-Support-with-VHS.md (hunk 2 above)
Comment (gkatsev) [inline source/_drafts/Inband-Captions-Support-with-VHS.md]: You can add another paragraph and embed your demuxed talk as a "general overview" thing.

File: source/_drafts/Inband-Captions-Support-with-VHS.md (hunk 1 above)
Comment (ldayananda) [reply source/_drafts/Inband-Captions-Support-with-VHS.md]: aha!

File: source/_drafts/Inband-Captions-Support-with-VHS.md (hunk 3 above)
Comment (ldayananda) [reply source/_drafts/Inband-Captions-Support-with-VHS.md]: I was actually referring to `traf -> tfhd -> trackId`, so yeah, `track` is probably better

Comment (gkatsev) [reply source/_drafts/Inband-Captions-Support-with-VHS.md]: ah, yeah, I think seeing `trak` would be a bit confusing unless it was brought up beforehand somewhere.

File: source/_drafts/Inband-Captions-Support-with-VHS.md (hunk 2 above)
Comment (gkatsev) [reply source/_drafts/Inband-Captions-Support-with-VHS.md]:
```
For a general overview, you can watch [my talk from demuxed](link)
{{embed youtube link via hexo somehow}}
```
Or something

Comment (gkatsev) [reply source/_drafts/Inband-Captions-Support-with-VHS.md]: https://hexo.io/docs/tag-plugins#YouTube

**Review by gkatsev — APPROVED** [review]
LGTM. The sample player can be embedded as well if we want.

**Comment by ldayananda** [comment]
Added the embed, let me know if that's a weird way to do it

**Pull request merged**